packages/mdc-textfield/foundation.js

```diff
@@ -213,8 +213,7 @@ class MDCTextFieldFoundation extends MDCFoundation {
       const isDense = this.adapter_.hasClass(cssClasses.DENSE);
       const labelScale = isDense ? numbers.DENSE_LABEL_SCALE : numbers.LABEL_SCALE;
       const labelWidth = this.adapter_.getLabelWidth() * labelScale;
-      const isRtl = this.adapter_.isRtl();
-      this.adapter_.notchOutline(labelWidth, isRtl);
+      this.adapter_.notchOutline(labelWidth);
```

Comment:
Are we able to completely remove the isRtl adapter API from text field and select now?